source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -99,12 +126,17 @@ class FilterConfig {
   Stats::Scope& scope_;
   Runtime::Loader& runtime_;
   Http::Context& http_context_;
-
+  // todo stop using pool as part of deprecating cluster scope stats.
```

Comment:
Super nit. I think the convention on placing TODO is `TODO(nezdolik):`

Reply:
@dio, think that `nezdolik` is not allowed word for spell check.

Reply:
pushed updates TODOs, waiting for spell check results